homeassistant/components/duotecno/entity.py

```diff
@@ -0,0 +1,36 @@
+"""Support for Velbus devices."""
+from __future__ import annotations
+
+from duotecno.unit import BaseUnit
+
+from homeassistant.helpers.entity import DeviceInfo, Entity
+
+from .const import DOMAIN
+
+
+class DuotecnoEntity(Entity):
+    """Representation of a Duotecno entity."""
+
+    _attr_should_poll: bool = False
+    _unit: BaseUnit
+
+    def __init__(self, unit) -> None:
+        """Initialize a Duotecno entity."""
+        self._unit = unit
+        self._attr_name = unit.get_name()
```

Comment:
For me it is not entirely clear what the difference between `unit.get_name()` and `unit.get_node_name()` is. Therefore I'm not sure if this is correct here.

Could you clarify the difference?

Reply:
the equipment is build as follows:

a node = physical device
a unit = basically a channel inside the node

so thats why the devices request the node name, and the entity the 'channel' name